### ISSUE TITLE
Get source code

### ISSUE THREAD

**Issue by kiquenet** [EpicGames/Signup#4]
I register my GitHub ID using your instructions and I cannot get source code.

**Comment by KASHYAPYAN** [comment]
Check your e-mail. 
Verify the link provided by Epic.

**Comment by codedajie** [comment]
‘Unreal Engine’ is invalidate now that signed into GitHub. Why

**Comment by Vickicup** [comment]
Same issue here - double-checked that the emails used to register for both are the same, to no avail. Some updates and assistance would be appreciated.

**Comment by NVKZ3843** [comment]
|Poor things :(
Only Vindovs download
Collect package will not give.
There are references both to collect the package, but there need a pact with gethaba with your personal account.
What to do?
Wait.
How many.

**Comment by NVKZ3843** [comment]
I beg your pardon:
Vindovs -> Windows

**Comment by lenfanson1** [comment]
that page is 404 always.

**Comment by NVKZ3843** [comment]
Hmm ... 404 always
So there Unity3D (the new version is free for students) or Panda.
All the engines are virtually identical.

**Comment by BadFactor** [comment]
U have to use the Epic Game Launcher. From here u can download all the stuff u need.

**Comment by NVKZ3843** [comment]
Thank you. I have already installed  Unreal Engine 4. 
Epic Game Launcher? Of course!

**Comment by carlovfx** [comment]
I linked my github name on the epicgames page and i cannot see anywhere here to download the source code. I also tried in the Epic Launcher but I cannot find it, what is going on??

**Comment by Phoenix616** [comment]
Worked perfectly fine for me, the invite to the EpicGames organisation arrived instantly.

**Comment by BadFactor** [comment]
@carlovfx maybe restart the launcher!
Or just wait a bit of time!

**Comment by FireLeaf** [comment]
github account name rather than github id?

**Comment by Crazymin** [comment]
I just wonder what this is 
![12](https://cloud.githubusercontent.com/assets/6031201/10513264/fbbea338-7378-11e5-8cb7-97bee18a85dd.PNG)

**Comment by NVKZ3843** [comment]
Crazymin : What do you not understand? 
it is a project. it seems so to me

**Comment by Crazymin** [comment]
Is it the Engine's source code?

**Comment by Phoenix616** [comment]
@Crazymin looks like it, yes. You need more then that to build it 'though.

Also for everyone having trouble getting access to the repos: Make sure you entered your correct GitHub username on your [account page](https://www.unrealengine.com/dashboard/settings).

**Comment by NVKZ3843** [comment]
@Crazymin Yes, but the process is not easy. And if you know C #, the wall of problems, and if not, alas: ((

**Comment by kkkon** [comment]
Worked perfectly fine for me.

I got the invitation that invite to the EpicGames Organization,
when display github.com web page logged in github.

But the invitation arrived after almost half hour,
when I link github account name at [unrealengine.com account setting](https://www.unrealengine.com/dashboard/settings)


'Unreal Engine' repo and 'Unreal Tournament' repo are 'github.com' private repository.
If you doesn't join EpicGames Organization, github.com show 404!

**Comment by c506928109** [comment]
that page is 404 always.

**Comment by cibergod** [comment]
nub

**Comment by cibergod** [comment]
Use Desktop application

**Comment by payamrastogi** [comment]
I register my GitHub ID using your instructions and I cannot get source code.

**Comment by NVKZ3843** [comment]
You have to wait a bit. I waited a day link.
Here's a link to my GitHub --> https://github.com/EpicGames/UnrealEngine
Good luck in game

**Comment by NVKZ3843** [comment]
Powerful engine.
I'm in shock. But we need to understand.
Thanks to developers.
